docs/commands.md

```diff
@@ -2578,16 +2578,16 @@ hal config canary signalfx account ACCOUNT [parameters] [subcommands]
  * `--no-validate`: (*Default*: `false`) Skip validation.
 
 #### Subcommands
- * `add`: Add a canary account to the SignalFx service integration.
- * `delete`: Delete a specific SignalFx canary account by name.
- * `edit`: Edit a canary account in the SignalFx service integration.
- * `get`: Get the specified canary account details for the SignalFx service integration.
- * `list`: List the canary account names for the SignalFx service integration.
+ * `add`: Add a canary account to the Signalfx service integration.
```

Comment:
Is the case change relevant to this change, or have we not regenerated the client for a while?

Reply:
Latter

Reply:
SG.